actions/checkForAndPlaceOrder.ts

```diff
@@ -118,38 +118,36 @@ const _checkForAndPlaceOrder: ActionFn = async (
         multicall,
         chainContext
       );
-      const error = pollError !== undefined;
+      const isError = pollResult.result !== PollResultCode.SUCCESS;
 
-      // Specific handling for each error
-      if (pollError) {
-        // Dont try again the same order
-        if (pollError.result === PollResultCode.DONT_TRY_AGAIN) {
-          ordersPendingDelete.push(conditionalOrder);
-        }
-
-        // TODO: Handle the other errors :) --> Store them in the registry and ignore blocks until the moment is right
-        //  TRY_ON_BLOCK
-        //  TRY_AT_EPOCH
+      // Don't try again the same order, in case thats the poll result
+      if (pollResult.result === PollResultCode.DONT_TRY_AGAIN) {
+        ordersPendingDelete.push(conditionalOrder);
       }
 
+      // Save poll result
+      conditionalOrder.pollResult = {
+        lastExecution: new Date(),
```

Comment:
We may want to be careful with the use of `Date()` here as multiple orders that are updated in the same block will all have different execution times. This may be inconsistent with expected logic.

Suggest that for `lastExecution` we reference the block.timestamp.

Reply:
That's a great suggestion. Actually, i can add also the block number